testing/test_kerns.py

```diff
@@ -3,7 +3,7 @@
 import tensorflow as tf
 import numpy as np
 import unittest
-from .reference import referenceRbfKernel, referencePeriodicKernel
+from testing.reference import referenceRbfKernel, referencePeriodicKernel
```

Comment:
Urggh, I just went down a long rabbit hole on stackoverflow on this. so long as it works with nosetests and the tests pass on the serves, I'm happy :)